debian/snapd.install

```diff
@@ -1,4 +1,5 @@
 /usr/bin/snap
+/usr/bin/snapctl
```

Comment:
Alright, I've put snapctl in `/usr/lib/snapd/util`, and added that directory to the PATH for both apps and hooks via `snap run`.